.gitignore

```diff
@@ -1,4 +1,5 @@
 cache/
 out/
 .vscode
-.idea
\ No newline at end of file
+.idea
+.DS_Store
```

Comment:
Recommend adding `.DS_Store` to your global `.gitignore` instead, since this is commonly generated on macOS

Reply:
Thanks!